doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -0,0 +1,173 @@
+#######################################
+RainForests calibration
+#######################################
+
+RainForests calibration is a situation dependent non-parametric method for the calibration
+of rainfall based on the ECPoint method of Hewson and Pillosu `Hewson & Pillosu, 2021`_.
+
+.. _Hewson & Pillosu, 2021: https://www.nature.com/articles/s43247-021-00185-9
+
+****************************************************
+Sub-grid variability as a means of calibration
+****************************************************
+
+ECPoint is based on the principle of using sub-grid variability as a means to calibrate
+grid-scale rainfall forecasts. Sub-grid variability in this context is the connection
+between the point observations one would measure within a grid-box when presented with
+a given areal average grid-scale forecast value. This relationship is encapsulated in the
+mapping that takes a grid-scale forecast value and maps it onto a distribution of typical
+point observation values, or equivalently a set of forecast error values, and is central
+to this calibration method.
+
+Naturally the relationship between this distribution and the forecast value is contingent
+on the underlying rainfall processes at play. For instance, the distribution associated
+with a rainband will be characteristically different to that associated with post-frontal
+showers and different still to that associated with deep-tropical convection. To this end,
+a suitable set of meteorological variables can be used to distinguish different rainfall
+regimes and identify the associated distribution that describes the underlying sub-grid
+variability.
+
+Equipped with the relevant distribution that describes the sub-grid variability for a given
+forecast, the grid-scale forecast value can be mapped onto a series of realisable point-scale
+values, yielding a form of conditional bias-correction. In the context of ensemble forecasts,
+this mapping is undertaken on a per realization basis to produce a series of realisable
+point-scale values which collectively form a calibrated ensemble, referred herein as a
+"super-ensemble".
+
+Hewson & Pillosu propose a framework (ECPoint) to determine and apply these distributions
+across a variety of weather types through the use of a manually tuned decision tree model
+taking an appropriate set of feature parameters (meteorological variables) as inputs.
+Details of this method can be found in `Hewson & Pillosu, 2021`_. 
+
+****************************
+The RainForests method
+****************************
+
+RainForests is an adaptation of the ECPoint method that seeks to use machine learning based
+tree models to replace the manually trained tree model of ECPoint. Here we use gradient-boosted
```

Comment:
Similar to above, I think "trained" gives the impression of some ML-like process, and it could be replaced with "built", "constructed" etc.

Reply:
Changed all instances of "manually trained" to "manually constructed".